### ISSUE TITLE
fcp container does not run 

### ISSUE THREAD

**Issue by xtophs** [kubernetes/federation#222]
Building the fcp container with `make quick-release`

```
+++ [0207 18:15:08] Syncing out of container
+++ [0207 18:15:13] Building tarball: src
+++ [0207 18:15:13] Building tarball: client linux-amd64
+++ [0207 18:15:13] Waiting on tarballs
+++ [0207 18:15:42] Building tarball: server linux-amd64
+++ [0207 18:15:42] Building fcp image for arch: amd64
+++ [0207 18:16:25] Deleting fcp image gcr.io/google_containers/fcp-amd64:v1.9.0-alpha.2.60_430416309f9e58-dirty
+++ [0207 18:17:04] Building tarball: final
+++ [0207 18:17:04] Building tarball: test
+++ [0207 18:17:15] Completed federation release build.
```

trying to run fcp container via `kubefed init` or `docker run` does not work. Container image does not appear to be valid.

federation apiserver fails with: 
```
kubectl logs apiserver-798b567969-6bndl apiserver --namespace=federation-system
container_linux.go:247: starting container process caused "exec: \"/fcp\": stat /fcp: no such file or directory"
```

Trying to run the container on the build machine fails as well.

```
$ docker import  _output/release-images/amd64/fcp-amd64.tar 
sha256:a344f4e7764c092783183b14a25d31a32a6b7f8df353e8b426190e7b2fd0a1d6
$ docker run -it a344f ls /
docker: Error response from daemon: invalid header field value "oci runtime error: container_linux.go:247: starting container process caused \"exec: \\\"ls\\\": executable file not found in $PATH\"\n".
```

Trying to import container from file system instead of the tarball results in the same error:
```
$ tar xvf ../_output/release-images/amd64/fcp-amd64.tar 
21097d87418fd37b93df7ed4212b7b8b10222f6feda04f2f16b6f573998ae55d/
21097d87418fd37b93df7ed4212b7b8b10222f6feda04f2f16b6f573998ae55d/VERSION
21097d87418fd37b93df7ed4212b7b8b10222f6feda04f2f16b6f573998ae55d/json
21097d87418fd37b93df7ed4212b7b8b10222f6feda04f2f16b6f573998ae55d/layer.tar
2fee09a140680be14fbb1890aafeb7e65792f4eac370251b8d63ecdcb5151b11/
2fee09a140680be14fbb1890aafeb7e65792f4eac370251b8d63ecdcb5151b11/VERSION
2fee09a140680be14fbb1890aafeb7e65792f4eac370251b8d63ecdcb5151b11/json
2fee09a140680be14fbb1890aafeb7e65792f4eac370251b8d63ecdcb5151b11/layer.tar
53f2f90db4320420f7da8f527e5f14439c15318482c8a112490de4e52f45164f.json
b2f6553d135e910021fb76c8e93059f13024aebf714f0ac537ef09fa02c34b27/
b2f6553d135e910021fb76c8e93059f13024aebf714f0ac537ef09fa02c34b27/VERSION
b2f6553d135e910021fb76c8e93059f13024aebf714f0ac537ef09fa02c34b27/json
b2f6553d135e910021fb76c8e93059f13024aebf714f0ac537ef09fa02c34b27/layer.tar
d0868bc098d76f12f2286a7af5ec1e7c2a78bb3d140a8767e93885a713923ea7/
d0868bc098d76f12f2286a7af5ec1e7c2a78bb3d140a8767e93885a713923ea7/VERSION
d0868bc098d76f12f2286a7af5ec1e7c2a78bb3d140a8767e93885a713923ea7/json
d0868bc098d76f12f2286a7af5ec1e7c2a78bb3d140a8767e93885a713923ea7/layer.tar
e2731de3eb5b9315656612804310de2b2faa76443bd0c03b10c32174ed547cbc/
e2731de3eb5b9315656612804310de2b2faa76443bd0c03b10c32174ed547cbc/VERSION
e2731de3eb5b9315656612804310de2b2faa76443bd0c03b10c32174ed547cbc/json
e2731de3eb5b9315656612804310de2b2faa76443bd0c03b10c32174ed547cbc/layer.tar
manifest.json
repositories
$ sudo tar -c . | docker import -
sha256:e75446b4309bdade844eb5dfc79cb145e720590054844f01ac7b8cab237da586
csc@fedbuild:/mnt/gopath/src/k8s.io/kubernetes/federation/xtoph$ docker run -it e7544 ls /
docker: Error response from daemon: invalid header field value "oci runtime error: container_linux.go:247: starting container process caused \"exec: \\\"ls\\\": executable file not found in $PATH\"\n".
```

Build box info:
```
$ uname -a
Linux fedbuild 4.13.0-1005-azure #7-Ubuntu SMP Mon Jan 8 21:37:36 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

$ docker version
Client:
 Version:      1.13.1
 API version:  1.26
 Go version:   go1.6.2
 Git commit:   092cba3
 Built:        Thu Nov  2 20:40:23 2017
 OS/Arch:      linux/amd64

Server:
 Version:      1.13.1
 API version:  1.26 (minimum version 1.12)
 Go version:   go1.6.2
 Git commit:   092cba3
 Built:        Thu Nov  2 20:40:23 2017
 OS/Arch:      linux/amd64
 Experimental: false
```

**Comment by mastamark** [comment]
As gleaned from another PR (https://github.com/kubernetes/federation/pull/186/files), this works for me from master of the federation repo and using a k8 1.9.1 cluster (in aws) as the host cluster:

cd $GOPATH/src/k8s.io/federation
make quick-release	# Will take awhile
docker load -i _output/release-images/amd64/fcp-amd64.tar
docker tag gcr.io/google_containers/fcp-amd64:<tag> <your dockerhub username>/fcp-amd64:<tag>
docker push <your dockerhub username>/fcp-amd64:<tag>
...
_output/dockerized/bin/linux/amd64/kubefed init myfed --host-cluster-context=... --image=<your dockerhub id>/fcp-amd64:<tag> ...
_output/dockerized/bin/linux/amd64/kubefed join ...
